verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/NodeTypeProvider.java

```diff
@@ -198,7 +199,10 @@ public Versioned<GraphQLType> createType(GraphQLContext context) {
 			nodeType.description(
 				"A Node is the basic building block for contents. Nodes can contain multiple language specific contents. These contents contain the fields with the actual content.");
 			interfaceTypeProvider.addCommonFields(nodeType, true);
-			nodeType.fields(createNodeInterfaceFields(context).forVersion(context));
+			List<GraphQLFieldDefinition> fields = createNodeInterfaceFields(context).forVersion(context);
```

Comment:
Here and below - create no `fields` type, if the target (micro)schema has no fields.